administrator/components/com_config/src/View/Component/HtmlView.php

```diff
@@ -82,9 +82,18 @@ public function display($tpl = null)
         $this->fieldsets   = $form ? $form->getFieldsets() : null;
         $this->formControl = $form ? $form->getFormControl() : null;
 
-        // Don't show permissions fieldset if not authorised.
-        if (!$user->authorise('core.admin', $component->option) && isset($this->fieldsets['permissions'])) {
-            unset($this->fieldsets['permissions']);
+        // Remove unauthorised preference tabs.
+        foreach($this->fieldsets as $key => $value) {
```

Comment:
```suggestion
        foreach ($this->fieldsets as $key => $value) {
```
Last PHPCS error, I think.